pkg/controllers/node/health/controller.go

```diff
@@ -126,7 +127,7 @@ func (c *Controller) Reconcile(ctx context.Context, node *corev1.Node) (reconcil
 	// The deletion timestamp has successfully been set for the Node, update relevant metrics.
 	log.FromContext(ctx).V(1).Info("deleting unhealthy node")
 	metrics.NodeClaimsDisruptedTotal.Inc(map[string]string{
-		metrics.ReasonLabel:       string(unhealthyNodeCondition.Type),
+		metrics.ReasonLabel:       strings.ToLower(string(unhealthyNodeCondition.Type)),
```

Comment:
Should we use snake case when we see a capital in the middle of the reason? e.g NodeNotReady becomes node_not_ready